gdc_client/download/client.py

```diff
@@ -281,22 +280,6 @@ def download_small_groups(self, smalls):
                 log.error("There are no files to download")
                 return [], 0
 
-            pbar = ProgressBar(
-                widgets=[
-                    Percentage(),
-                    " ",
-                    Bar(marker="#", left="[", right="]"),
-                    " ",
-                    ETA(),
-                    " ",
-                    FileTransferSpeed(),
-                    " ",
-                ],
-                maxval=1,
-                fd=sys.stdout,
-            )
-            pbar.start()
-
```

Comment:
so is this download case for pbar completely covered by the one in gdc_client/parcel/segment.py?

Reply:
this progress bar was only to show the progress of grouping small files, I'm gonna push update to display the progress bar for the actual download

Reply:
done